lms/djangoapps/verify_student/management/commands/tests/test_send_verification_expiry_email.py

```diff
@@ -101,6 +101,8 @@ def test_most_recent_verification(self):
         outdated_verification.status = 'approved'
         outdated_verification.save()
 
+        call_command('send_verification_expiry_email')
+
```

Comment:
As far as I can tell, this test was just bogus before, because it forgot to actually call the command inbetween the setup code and the assert code.  :man_shrugging: